packages/ember-glimmer/tests/integration/event-dispatcher-test.js

```diff
@@ -138,6 +139,34 @@ moduleFor('EventDispatcher#setup', class extends RenderingTest {
     this.$('div').trigger('myevent');
   }
 
+  ['@test canDispatchToEventManager is deprecated'](assert) {
+    this.dispatcher.canDispatchToEventManager = null;
```

Comment:
In theory, we should not need to force this to be `null` in this test, the default of `undefined` _should_ be enough